reference-infrastructure/README.md

```diff
@@ -0,0 +1,33 @@
+# Reference infrastructure
+
+For full production deployment you will need to provide some common components that
+are then passed as values to the helm chart when installing. These components are:
+
+* Shared storage
+* Database
+* Elasticsearch (optional)
+
+## Shared storage
+
+### Cloud managed shared storage
+
+#### Requirements
+
+#### Example
+
+
+### NFS server - Bitbucket DC requirement
```

Comment:
Hi @tan-ro it would be great if you could give the wording a here a review.

Reply:
@dylanrathbone Looks good to me.